packages/aws-cdk-lib/pipelines/lib/codepipeline/codepipeline-source.ts

```diff
@@ -102,6 +102,7 @@ export abstract class CodePipelineSource extends Step implements ICodePipelineAc
    * `codeBuildCloneOutput`.
    *
    * @param repoString A string that encodes owner and repository separated by a slash (e.g. 'owner/repo').
+   * Note: The provided string must be resolvable at runtime. Unresolved tokens will lead to runtime errors.
```

Comment:
@Nikola-Milovic I don't like adding this between `@params`. We do some weird parsing in jsii and this could ruin the [brittle] parser. Let's keep it on the same line, or show me a screenshot of what the rendered docstring looks like to prove to me that it's rendered properly.

Reply:
Yeah, I agree that it could cause a problem. In VSCode and Neovim it looks fine, but even in those two, there is a discrepancy as to how the output from the LSP is shown. VSCode puts it all in the same line, while Neovim retains the new line. 

What do you think of the new version? Removed the extra fluff sentence and made it continuous.

A side question, how do you avoid having really long lines for these `@params` if a newline can mess up the jsii parser?

Reply:
I'm not sure if it does mess it up. If it looks fine in VSCode when you hover over the prop then I think its fine. I was asking for a screenshot of that if you wanted to keep them on separate lines, since I am lazy and do not want to check out your branch just to check :)